### ISSUE TITLE
Little mistake - Get Started Website Description

### ISSUE THREAD

**Issue by Midnight93** [facebookresearch/pytorchvideo#53]
Link: https://pytorchvideo.org/

My import libraries

```python
import torch
from torchvision.transforms import Compose, Lambda
from torchvision.transforms._transforms_video import (
    CenterCropVideo,
    NormalizeVideo,
)
from pytorchvideo.data.encoded_video import EncodedVideo
from pytorchvideo.transforms import (
    ApplyTransformToKey,
    ShortSideScale,
    UniformTemporalSubsample,
    UniformCropVideo
)
```
Current version on website:

```python
# Generate top 5 predictions
post_act = F.softmax(dim=1)
preds = post_act(preds)
pred_class_ids = preds.topk(k=5).indices
```

Correct version:

```python
# Generate top 5 predictions
post_act =torch.nn.Softmax(dim=1)
preds = post_act(preds)
pred_class_ids = preds.topk(k=5).indices
print(pred_class_ids)

#Mapping predicted classes

pred_class_names = [kinetics_id_to_classname[int(i)] for i in pred_classes[0]]
print("Predicted labels: %s" % ", ".join(pred_class_names))

```
## Output example

```bash
[1] tensor([[192, 104, 315, 268,  78]], device='cuda:0')
[2] Predicted labels: marching, driving tractor, sled dog racing, riding camel, crossing river
```

### Documentation 

https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html

**Comment by kalyanvasudev** [comment]
Thanks for the catch. Fixing it.
